examples/running-tests-using-travis-and-saucelabs/tests/index-test.js

```diff
@@ -0,0 +1,8 @@
+fixture `Getting Started`
+    .page('http://localhost:9090/index.html');
+
+test('My test', async t => {
+    await t
```

Comment:
Can we have something not as specific as native dialogs in example? Maybe some button which makes greeting visible when clicked (add assertion for visible text).